sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/UnsupportedOperationsSuite.scala

```diff
@@ -129,6 +156,18 @@ class UnsupportedOperationsSuite extends SparkFunSuite {
     outputMode = Complete,
     expectedMsgs = Seq("(map/flatMap)GroupsWithState"))
 
+  // Deduplication:  Not supported after a streaming aggregation
```

Comment:
actually can you add a test for both, dropdup and mapgroupsWithstate, that tests that these operations is allowed on a batch subplan inside a streaming plan.